test/rest.test.js

```diff
@@ -127,7 +127,7 @@ describe('strong-remoting-rest', function() {
     });
 
     it('should disable stack trace', function(done) {
-      objects.options.errorHandler.disableStackTrace = true;
+      objects.options.strongErrorHandler.debug = false;
```

Comment:
Ditto, this line should not need to be changed, the old API should continue to work in the new version.

If you prefer, then we can support both settings `debug:false` and `disableStackTrace:true`, but it's important to ensure that `disableStackTrace` keeps working.

However, while you are making this change, please change the default value of `disableStackTrace` to `true`, so that apps are configured for security by default.